pyomo/neos/tests/test_neos.py

```diff
@@ -236,28 +236,46 @@ class TestSolvers_direct_call_min(RunAllNEOSSolvers, DirectDriver,
                                   unittest.TestCase):
     sense = pyo.minimize
 
-    # Add the CBC test to the nightly suite
-    @unittest.category('nightly')
-    def test_cbc(self):
+    # Add the CBC test to the nightly suite, but with a non-fatal
+    # (short) timeout
+    #
+    # TODO: remove queued job from NEOS servers.  Using timeout() leaves
```

Comment:
Should we create a new issue for this TODO?

Reply:
I do not have a strong opinion: it is not a high priority task, and will likely be ignored until we get around to (re)thinking the entire SolverManager system.  An issue would very likely end up sitting there for a long time.